docs/content/integrations/dbt/reference.mdx

```diff
@@ -170,8 +223,14 @@ In your CI/CD workflows for your Dagster project:
 
 1. Include any secrets that are required by your dbt project in your CI/CD environment.
 2. Clone the dbt project repository as a subdirectory of your Dagster project.
-3. Run `dbt deps` to build your dbt project's dependencies.
-4. Run `dbt parse` to create a dbt manifest for your Dagster project.
+3. Depending on whether <PyObject object="DbtCliProject" module="dagster_dbt" /> is used, this step will vary:
+   - **For projects using `DbtProject`**, run `dagster-dbt project prepare-and-package --file path/to/project.py` to
+     - Build your dbt project's dependencies,
+     - Create a dbt manifest for your Dagster project, and
+     - Package your dbt project
+   - **For projects not using `DbtProject`**:
```

Comment:
Should this be `For projects using DbtCliResource`, since the previous step only gives the user two options: `DbtProject` or `DbtCliResource`?

Reply:
Good catch! Updated in [dcd6d90](https://github.com/dagster-io/dagster/pull/23012/commits/dcd6d90778b9495ed484736f7e2d1ea3eb54ff3c)